core/src/com/unciv/logic/GameSaver.kt

```diff
@@ -107,6 +128,15 @@ object GameSaver {
     }
 
     fun autoSaveSingleThreaded (gameInfo: GameInfo) {
+        // If the user has chosen a custom save location outside of the usual game directories,
+        // they'll probably expect us to overwrite that instead. E.g. if the user is saving their
+        // game to their Google Drive folder, they'll probably want that progress to be synced to
+        // other devices automatically so they don't have to remember to manually save before
+        // exiting the game.
+        gameInfo.customSaveLocation?.let {
+            saveGame(gameInfo,gameInfo.currentPlayer + " -  " + gameInfo.turns + " turns", false)
```

Comment:
Does this mean that after 100 turns I'll have 100 autosave files in my folder? I don't think that's what we're aiming for.
Also, when loading a game, do I have an option to hide the previous autosaves? Because if not then this too could get annoying (I have 3 games, and 30 autosave files - need to 'search' for the right file)

Reply:
> Does this mean that after 100 turns I'll have 100 autosave files in my folder?

Nope. Unfortunately for games saved to custom locations, we have to overwrite the file for autosave. This is a restriction put in place by Android, since we only get access to the file that's loaded and not the folder. As I stated in the last paragraph of the PR and in Discord, opening the folder on Android doesn't work for cloud storage providers at all, making it basically no different than simply using external storage.

Reply:
I replaced the parameter with an empty string to avoid confusion